core/modules/admin_bar/css/admin_bar.css

```
@@ -166,6 +166,20 @@
   padding-right: 28px;
 }
 
+/* Locale link in Admin bar */
+#admin-bar .dropdown .admin-bar-locale-links > span {
+  background: transparent url(../images/language--white--64.png) 95% center no-repeat;
+  background-size: 20px;
+  padding-right: 30px;
+  /* LTR */
```

AFAIK, the `/* LTR */` indication is added next to the specific CSS property(ies):

```suggestion
  padding-right: 30px; /* LTR */
```

Does PHPCS complain if we do it that way?